ml/notebooks/netdata_anomaly_detection_deepdive.ipynb

```diff
@@ -0,0 +1,1705 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    # initialize dictionary for storing anomaly bits for each dimension
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netdata/netdata/pull/12313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>